deliverables/README.md

```diff
@@ -8,61 +11,134 @@ To answer questions, please just put the answer on the next line after the quest
 
 Example:  
 
-**What is your name?**  
-Sir Student of Ruby
+**What is your name?** <br/> 
+Chloe Wolf
 
-**What is your quest?**
-I seek the Rails  
+**What is your quest?**<br/>
+To earn a serious summer internship that will hopefully lead to multiple career opportunities post-graduation.   
 
-**What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+**What is your favorite color?**<br/>
+Mine really is Green.  
 
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
+
+**What is the role of HTML in a web page?** <br/>
+HTML is what creates the structure of the web page. 
+
+
+
+**What is the role of CSS in a web page?** <br/>
+CSS controls the aesthetics of the website, like 
 What is the role of Javascript in a web page?  
 
 ---
 
 ### Section 2: HTML and CSS 101
-
 #### meta
-Why do we separate HTML and CSS?  
+**Why do we separate HTML and CSS?**  
+With a separate CSS file you can apply that style to multiple HTML pages so that they are using all of the same CSS. This comes in handy when you want to change something in CSS for multiple webpages.
+
 
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+**What are classes and IDs (and how are they different)?**  
+IDs can be used to identify specific elements in HTML. Classes are made in CSS and when applied to elements in HTML whatever you made in CSS will apply. Each element can only have one ID but you can put as many classes as you want in an element. 
+
+
+**What are elements?**  
+elements make up the web page, tags mark the beginning and end of an element.
+
+**What are tags?**   
+tags help make up HTML, they surround elements and apply meaning to it. 
+
+
+**What are attributes?**   
+attributes are additional information that go in between the tags of an element, these are in quotations and are often values.
+
+
+**What are forms?**  
+Forms define a form that allows user inputted data to be sent somewhere.
+
+
+**What is a div?**  
+divs are used to add extra information to an existing chunk of information in HTML. A div can also be used to group existing sections of code together.
```

Comment:
Generally just used a way to structure and group code on the page like you said.